### PR TITLE
Documentation how to use alternative PHP versions

### PR DESCRIPTION
**Pull request by martijneichelsheim**

### CODE REVIEW

File: docs/hypernode-platform/php/how-to-use-multiple-php-versions.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+myst:
+  html_meta:
+    description: 'Sometimes you want to use multiple PHP versions to test a Magento update as Example. 
+    Find out how to enable and use alternative PHP versions. '
+    title: How to use Alternative PHP versions? | Hypernode
+redirect_from:
```
Comment (tdgroot) [inline docs/hypernode-platform/php/how-to-use-multiple-php-versions.md:7]: redirect_from is not necessary for new docs

File: docs/hypernode-platform/php/how-to-use-multiple-php-versions.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  - /en/hypernode/php/how-to-use-alternative-php-versions/
+---
+
+<!-- source: https://support.hypernode.com/en/hypernode/php/how-to-use-alternative-php-versions/ -->
```
Comment (tdgroot) [inline docs/hypernode-platform/php/how-to-use-multiple-php-versions.md:11]: Not necessary

**Review by burak-tb — APPROVED** [review]
lgtm

**Review by ikidnapmyself — APPROVED** [review]
lgtm -- @burak-tb is co-pilot account that i never use

**Review by tdgroot — APPROVED** [review]

**Pull request merged**